test/loadTest/cache_refresh/README.md

```diff
@@ -74,6 +74,16 @@ These tests are used to verify the cache refresh in a balanced system.
      -u show the usage [OPTIONAL]
   ```
   
+  Dependency:
+  ```
+  jq is a lightweight and flexible command-line JSON processor. It is used for the json with statistics returned from ContextBroker.
+       steps to install jq:
+      ---------------------
+           wget http://stedolan.github.io/jq/download/linux64/jq
+           chmod +x ./jq
+           cp jq /usr/bin
```

Comment:
Locally installed stuff should be at /usr/local/bin. In addition, you need root privileges to copy there, so probably is better to say:

```
sudo cp jq /usr/local/bin
```